server/api/main.py

```diff
@@ -146,13 +146,24 @@ async def http_status_error_handler(
         request_id = request.state.request_id
     status_code = exc.response.status_code
     error_message = repr(exc)
-    logger.warning(
-        "Request handling returned error status",
-        error_message=error_message,
-        status_code=status_code,
-        traceback=traceback.format_exc(),
-        request_id=request_id,
-    )
+    log_message = "Request handling returned error status"
+
+    if type(exc) in mlrun.errors.EXPECTED_ERRORS:
```

Comment:
I think the idiomatic way to do this is
```suggestion
    if isinstance(exc, mlrun.errors.EXPECTED_ERRORS):
```
or
```suggestion
    if isinstance(exc, tuple(mlrun.errors.EXPECTED_ERRORS)):
```
if we don't want to change `mlrun.errors.EXPECTED_ERRORS` to a tuple.

Reply:
oh sure I can make it a tuple